sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/__init__.py

```diff
@@ -33,69 +36,31 @@ class BaseExporter:
     """
 
     def __init__(self, **options):
-        self._telemetry_processors = []
-        self.options = ExporterOptions(**options)
-        retry_policy = RetryPolicy(timeout=self.options.timeout)
-        proxy_policy = ProxyPolicy(proxies=self.options.proxies)
+        options = ExporterOptions(**options)
```

Comment:
Would this work if `ExporterOptions` is not a `BaseObject` anymore? I know we only have one configuration now, but in case in the future we do add more, I would keep it as a `BaseObject`.